src/xcode/ENA/ENA/Source/Services/TicketValidation/AllowList/TicketValidationAllowList.swift

```diff
@@ -3,6 +3,7 @@
 //
 
 import Foundation
+import ENASecurity
```

Comment:
Same here, no other changes

Reply:
the ValidationServiceAllowlistEntry struct is moved into the security package as it is needed for the Trust Evaluation class